### ISSUE TITLE
Cant view outputs from task view, again.

### ISSUE THREAD

**Issue by ascobie** [Azure/BatchExplorer#2018]
Latest Insider

Cannot view an output image from the task node output view

![image](https://user-images.githubusercontent.com/10934358/56100906-af17e500-5f72-11e9-8458-60ab44d7cc21.png)

You can from the Data menu. I thought this was fixed a while back, might have regressed again.

**Comment by timotheeguerin** [comment]
Are you on stable or insider?

**Comment by ascobie** [comment]
it's insider latest, and i can view the image in the Data explorer. 
though when i restarted BEX last night i got another update, so will check again.

**Comment by ascobie** [comment]
Still failing.

![image](https://user-images.githubusercontent.com/10934358/56256907-b11ea700-611e-11e9-8ebc-1e1c3c471785.png)

still works from the data menu.

![image](https://user-images.githubusercontent.com/10934358/56256934-c72c6780-611e-11e9-8308-350a89082dcd.png)

No errors in the console.

**Comment by timotheeguerin** [comment]
Yeah, maybe the img is added before we are done downloading as you say and chromium doesn't reload if the local file change
